helm/fluentd-elasticsearch/templates/configmaps.yaml

```diff
@@ -253,7 +253,7 @@ data:
     # statements, such as those that include entire object bodies, get split into
     # multiple lines by glog.
     # Example:
-    # I0204 07:32:30.020537    3368 server.go:1048] POST /stats/container/: (13.972191ms) 200 [[Go-http-client/1.1] 10.244.1.3:40537]
+    # [[ `I0204 07:32:30.020537    3368 server.go:1048] POST /stats/container/: (13.972191ms) 200 [[Go-http-client/1.1] 10.244.1.3:40537]` ]]
```

Comment:
Here escaping was not necessary because there is no `]]` closing sequence. But for sanity I added escaping here as well.